openassessment/xblock/openassessmentblock.py

```diff
@@ -364,6 +364,82 @@ def student_view(self, context=None):
         context = Context(context_dict)
         fragment = Fragment(template.render(context))
 
+        return self._update_and_return_fragment(fragment, 'OpenAssessmentBlock')
```

Comment:
Nit: IMO these calls would be easier to read if the initialize func were passed as a keyword,

e.g. ``return self._update_and_return_fragment(fragment, initialize_js_func='OpenAssessmentBlock')``